torch/csrc/distributed/c10d/logging.h

```diff
@@ -10,11 +10,18 @@
 
 #include <c10/util/Logging.h>
 
+namespace torch {
```

Comment:
Nit: do you mind changing the namespace to `c10d::detail` and naming the function `vformat`?

```
namespace c10d {
namespace detail {

template <typename... T>
std::string vformat(fmt::string_view fmt, T&&... args) {
  return fmt::vformat(fmt, fmt::make_format_args(std::forward<Args>(args)...));
}

} // namespace detail
} // namespace c10d
```

Reply:
Done.